src/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+add_library(asteroid-hrm main.cpp resources.qrc)
+set_target_properties(asteroid-hrm PROPERTIES PREFIX "")
```

Comment:
`set_target_properties(asteroid-hrm PROPERTIES PREFIX "" SUFFIX "")` ?

Sorry for the bikeshedding... :p otherwise the series looks great and I think is ready for merging (once meta-asteroid is done)